docker/scripts/install_R_version

```diff
@@ -60,54 +60,62 @@ activate_R_${R_VERSION}
 
 function checkRInstallLog() {
   log_file=$1
-  if [ grep -E '(packages .* are not available)|(installation of package .* had non-zero exit status)' $log_file ]; then
+  if grep -E '(packages .* are not available)|(installation of package .* had non-zero exit status)' $log_file; then
     echo "Errors detected in R package installation."
     exit 1
   fi
 }
 
 # Install dependencies
 echo "Installing dependencies for R ${R_VERSION}"
-/usr/local/R/current/bin/R -e "chooseCRANmirror(graphics=FALSE, ind=58); \
+/usr/local/R/current/bin/R -e "chooseCRANmirror(graphics=FALSE, ind=1); \
   install.packages(c( \
-    'dplyr', 'text2vec', 'R.utils', 'AUC', 'mlbench', 'flexclust', 'randomForest', \
+    'dplyr', 'R.utils', 'AUC', 'mlbench', 'flexclust', 'randomForest', \
     'bit64', 'HDtweedie', 'jsonlite', 'statmod', 'devtools', 'roxygen2', 'testthat', 'Rcpp', 'fpc', 'RUnit', \
-    'ade4', 'e1071', 'ggplot2', 'LiblineaR', 'optparse', 'uuid', 'usethis', 'rgl', 'plot3Drgl' \
+    'ade4', 'e1071', 'ggplot2', 'LiblineaR', 'optparse', 'uuid', 'usethis', 'rgl', 'plot3Drgl',  \
+    'IRdisplay', 'plotly' \
   )) \
 " 2>&1 | tee R.install.1.log
 checkRInstallLog R.install.1.log
-# Latest XGBoost (from source)
-/usr/local/R/current/bin/R -e "chooseCRANmirror(graphics=FALSE, ind=58); \
-  install.packages(c('xgboost'), type='source') \
+# Latest XGBoost (from source) requires c++14, possibly causing troubles
```

Comment:
@honzasterba It might not be related to C++14 but was the only obvious change between the two versions.

I'm not sure if it is OK not using the latest version but the latest doesn't compile and I didn't have much time to explore if I can upgrade gcc etc.

Reply:
please create a JIRA for this

Reply:
Done. https://0xdata.atlassian.net/browse/PUBDEV-7812